layouts/partials/form-section.html

```diff
@@ -0,0 +1,35 @@
+<section id="{{ .sectionId }}" class="py-8 bg-gray-200 px-4 lg:px-0">
+    <div class="container mx-auto max-w-6xl py-8 sm:px-10 text-sm relative">
+        <div class="mb-8 md:flex md:my-8 md:max-w-32 align-top">
+            <div class="md:w-5/12 md:pt-10">
+                <h4 class="text-black">
+                    {{ .quoteTitle }}
+                </h4>
+                <div class="relative py-8 px-10">
+                    <i class="fa fa-quote-left text-white absolute top-0 left-0 ml-4 mt-4 text-5xl z-0"></i>
+                    <p class="leading-loose mt-0 z-10 relative text-gray-800 text-sm">
+                        {{ .customerQuote }}
+                    </p>
+                    <div class="leading-relaxed flex items-start">
+                        <div>
+                            <div class="text-blue-700 font-semibold text-sm">
+                                {{ .quoteName }}
+                            </div>
+                            <div class="text-blue-500">
+                                {{ .quoteJobTitle }}
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="mt-4 mx-auto md:w-7/12 md:ml-8 md:mt-0 p-10 bg-white rounded rounded-lg shadow-lg border border-gray-300">
+                <h3 class="text-center">{{ .formHeader }}</h3>
+                <div class="mx-auto flex justify-center">
+                    <div class="hs-form hs-form-fg-dark font-normal">
+                        {{ partial "hubspot-form.html" (dict "hubspotFormID" .hubspotFormID "classNames" .formClassNames) }}
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
```

Comment:
Newline. 😄 (If you're using VSCode, you can set this with `"files.insertFinalNewline": true`.)